<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RenameTV</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listEpisodes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>121</y>
      <width>531</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TV Show name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSearch">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
